packages/components/icons/README.md

```diff
@@ -105,3 +105,34 @@ const app = () => <LeadingIcon icon={<ExportIcon />} />;
 ### Where to use
 
 This component can be used wherever it is necessary to display a themed icon.
+
+## Custom Icon
+
+The custom icon is a an eye-catching visual element that should be used when an additional visual prominence is needed for displaying a custom SVG icon in the UI.
+
+The component is exported as a separate entry point:
+
+```js
+import CustomIcon from '@commercetools-uikit/icons/custom-icon';
+```
+
+### Usage
+
+```js
+import CustomIcon from '@commercetools-uikit/icons/custom-icon';
+import { YourCustomIcon } from './your-custom-icon-directory';
+
+const app = () => <Icon icon={<YourCustomIcon />} />;
+```
+
+### Properties
+
+| Props       | Type                                                      | Required | Values                                              | Default | Description                                     |
+| ----------- | --------------------------------------------------------- | :------: | --------------------------------------------------- | ------- | ----------------------------------------------- |
+| `size`      | `string`                                                  |          | '10', '20', '30', '40'                              | '20'    | Specifies the icon size                         |
+| `icon`      | `union`<br/>Possible values:<br/>`, ReactElement, string` |    -     | A `ReactNode` or `string` that display a custom SVG |         | Icon displayed as a child of this component     |
+| `hasBorder` | `boolean`                                                 |          | `true`, `false`                                     | `false` | Specifies whether the element displays a border |
+
+### Where to use
```

Comment:
Not sure if we need both the description and this section.

@FilPob What do you think?

Reply:
not sure what is meant here. (too much code for my eyes :D ) Are there two descriptions somewhere? I would probably suggest to follow whatever approach is there in other components

Reply:
We currently have a description of the component and where to use it:

![Screenshot 2024-05-07 at 15 32 29](https://github.com/commercetools/ui-kit/assets/97907068/6eab2c1f-d9da-4827-9573-64516fdd852c)

Does it make sense to have both. If so, what do you suggest to explain in each section?

Reply:
ah okay got it... No, the first description is rather targeted for the Leading icon. The second fits better I think

Reply:
You mean it's ok to not include the "Where to use" section and just keep the initial component description but changing it to the one I suggested, don't you?

> This component is meant to be used whenever consumers need to render an icon which is not part of the [ui-kit icon set](https://uikit.commercetools.com/?path=/story/components-icons--all-icons).
In order to keep visual consistency, we want to keep the available sizes of all icons equal.
Bear in mind we would expect custom SVG icons to not contain size attributes so it can be controlled based on the components size attribute.

Reply:
yes exactly :)

Reply:
hey @CarlosCortizasCT would you like me to make these changes?

Reply:
Yes, please

Reply:
PR is updated to include the requested wording in the description and removes the "where to use" section as specified